contrib/CMakeLists.txt

```diff
@@ -275,12 +275,18 @@ if (USE_INTERNAL_AWS_S3_LIBRARY)
     set (CMAKE_CMAKE_REQUIRED_INCLUDES ${save_CMAKE_REQUIRED_INCLUDES})
     set (CMAKE_REQUIRED_FLAGS ${save_CMAKE_REQUIRED_FLAGS})
     set (CMAKE_CMAKE_MODULE_PATH ${save_CMAKE_MODULE_PATH})
+
+    # The library is large - avoid bloat.
+    target_compile_options (curl PRIVATE -g0)
+endif ()
+
+if (USE_INTERNAL_AWS_S3_LIBRARY)
     add_subdirectory(aws-s3-cmake)
 
     # The library is large - avoid bloat.
     target_compile_options (aws_s3 PRIVATE -g0)
     target_compile_options (aws_s3_checksums PRIVATE -g0)
-    target_compile_options (curl PRIVATE -g0)
```

Comment:
What's wrong?

Reply:
> What's wrong?

@alexey-milovidov nothing, this line is moved to a new place above